src/mariner/index.ts

```diff
@@ -2,4 +2,4 @@ export { DependencyDetailsRetriever } from '../dependency-details-retriever';
 export { readConfigFile, Config } from '../config';
 export { Issue, IssueFinder } from '../issueFinder';
 export { Logger, getLogger, setLogger } from '../tab-level-logger';
-export { generateConfluenceMarkdown } from '../Utilities/generateConfluenceMarkdown';
+export { generateConfluenceMarkdown, cleanMarkdown } from '../Utilities/generateConfluenceMarkdown';
```

Comment:
Oh crap. We don't want mariner to export that. Sorry!

The test should import generateConfluenceMarkdown from mariner, but should import cleanMarkdown directly from the source file as you had it.